libs/pxt-common/pxt-helpers.ts

```diff
@@ -402,6 +402,22 @@ namespace helpers {
         return stringSubstrHelper(s, start, end - start);
     }
 
+    // TODO move to PXT
```

Comment:
```suggestion
```

I see this is duplicated from tolowercase / was accidentally left in when that was originally moved into pxt, but might as well drop the errant comments now ~